core/templates/dev/head/domain/exploration/StateTopAnswersStatsBackendApiServiceSpec.js

```diff
@@ -0,0 +1,62 @@
+// Copyright 2018 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Unit tests for StateTopAnswersStatsBackendApiService.
+ */
+
+describe('StateTopAnswersStatsBackendApiService', function() {
+  beforeEach(module('oppia'));
+  beforeEach(inject(function($injector) {
+    this.StateTopAnswersStatsBackendApiService =
+      $injector.get('StateTopAnswersStatsBackendApiService');
+  }));
+
+  beforeEach(inject(function($injector) {
```

Comment:
I think you can combine this with the previous beforeEach?

Reply:
Done.